spec/request/caregivers_assistance_claims_request_spec.rb

```diff
@@ -80,7 +80,7 @@
     context 'when unauthenticated' do
       it_behaves_like 'any invalid submission'
 
-      timestamp = Date.parse('2020-03-09T06:48:59-04:00')
+      timestamp = DateTime.parse('2020-03-09T06:48:59-04:00')
```

Comment:
Why was this changed?  Should it be the Railsy `Time.zone`?

Reply:
This change was just to retain the time information all together. I realized this test was comparing a `Date` to `DateTime`.

Reply:
This change was needed for this following expectation:
```
expect(DateTime.parse(res_body['data']['attributes']['submittedAt'])).to eq timestamp
```

The test would pass before because I was using `CARMA::Client#create_submission_stub`, which generates a DateTime of `#now`. This would make the test above pass because it would compare a `Date` with time 0 to a `DateTime` with time 0.

But once I stopped using the stub, and started using the VCR cassette, the time info needed. Because I was then comparing `Date` with time 0 and `DateTime` with time `06:48:59-04:00`.